static/js/components/inputs/CountrySelectField.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import _ from 'lodash';
+import iso3166 from 'iso-3166-2';
+import SelectField from './SelectField';
+
+let countryOptions = _(iso3166.data)
+  .map((country_info_obj, country_code) => ({
+    value: country_code,
+    label: country_info_obj.name
+  }))
+  .sortBy('label').value();
```

Comment:
the previous one i agree with, but in this case, we use _.sortBy immediately after the map. to me it makes a lot more sense to use _ and chain them